marlowe-playground-client/src/Home.purs

```diff
@@ -26,70 +26,90 @@ render state =
             , p_ [ text "Marlowe contracts can be built in different ways. You can write them as Marlowe text, but also use the Blockly visual programming tool to create contracts by fitting together blocks that represent the different components. Marlowe is embedded in JavaScript and Haskell, and so you can use features from them to help you to build Marlowe contracts more readably and succinctly." ]
             ]
         ]
+    , hr_
     , div [ classes [ flex, ClassName "start-with-container" ] ]
-        [ div [ classes [ fullWidth, flex ] ]
-            [ div [ class_ (ClassName "even-item") ] []
-            , startWithHaskell state
-            , div [ class_ (ClassName "even-item") ] []
-            ]
+        [ startWith 1 "start with Haskell" startWithHaskell
+        , startWith 2 "start with Javascript" startWithJavascript
+        , startWith 3 "start with Marlowe or Blockly" startWithMarlowe
```

Comment:
I'm dying to `mapWithIndex` this.

Reply:
If you refactored it, you'd get to:

``` purescript
[ startWith 1 Haskell
, startWith 2 JavaScript
, startWith 3 Marlowe
]
```

Which IMO is closer to what you want to say.

Reply:
you are missing the rendering function though, you would need an additional function to wrap that up with a case expression. Then it seems to be getting complicated.